cloudbuild.yaml

```diff
@@ -0,0 +1,15 @@
+steps:
+  - name: "gcr.io/cloud-builders/docker"
+    args: ["build", "-t", "gcr.io/$PROJECT_ID/greencheck", "."]
+  - name: "docker/compose:1.23.2"
```

Comment:
is there a way to somehow cache the docker image so it doesn't need to be rebuild unless the Dockerfile changed? Probalby not as you need to copy the files into the image to let it do it's magic right?

Reply:
I think so, but I'd prefer to make a separate issue, as now we have the CI hardness, making the build go faster once this is merged in sounds like the kind of issue that someone without too much context about the how the everything works could pick up.